### PR TITLE
Replace Sarah by Wolfman on code examples

### PR DESCRIPTION
**Pull request by r-gaia-cs**
We start the lesson talking about Dracula and Wolfman collaboration so make sense to replace Sarah with Wolfman.

### CODE REVIEW

File: 03-conflict.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ $ cat mars.txt
 Cold and dry, but everything is my favorite color
 The two moons may be a problem for Wolfman
 But the Mummy will appreciate the lack of humidity
-This line added to Sarah's copy
+This line added to Wolfram's copy
```
Comment (jiffyclub) [inline 03-conflict.md]: Wolfram -> Wolfman

Comment (r-gaia-cs) [inline 03-conflict.md]: Sorry for the type. I fixed it.

**Comment by jiffyclub** [comment]
Looks like the HTML is out of date now.

**Comment by r-gaia-cs** [comment]
> Looks like the HTML is out of date now.

Sorry about that. Fixed.

**Pull request merged**